src/test/java/org/reaktivity/nukleus/maven/plugin/internal/generated/ListWithArrayFWTest.java

```diff
@@ -280,8 +280,18 @@ private static StringFW asStringFW(
         }
     }
 
-    private static List<StringFW> asStringFWArray(StringFW... values)
+    private static VariantArrayFW<VariantEnumKindWithString32FW> asVariantArrayFW()
```

Comment:
Suggest passing in the list of `StringFW` items that should be converted to `VariantArrayFW` instead of hardcoding those values in this method.